src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/CryptoSignedXmlRecursionException.cs

```diff
@@ -13,17 +13,13 @@ namespace System.Security.Cryptography.Xml
     /// hash. ComputeHash can throw different kind of exceptions.
     /// This unique exception helps catch the recursion limit issue.
     /// </summary>
-    internal class CryptoSignedXmlRecursionException : XmlException
+    [Serializable]
+    [System.Runtime.CompilerServices.TypeForwardedFrom("System.Security, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a")]
+    public class CryptoSignedXmlRecursionException : XmlException
```

Comment:
Why is this public now?

Reply:
System.Cryptography.Xml does not expose internals with the InternalsVisibleToAttribute, therefore to successfully typeforward from System.Security to this type we need to make it public in the src assembly.